### PR TITLE
Expose editor "blue button" as menu

### PR DESCRIPTION
**Pull request by jrieken**
* rename proposed `MergeToolbar` to `EditorContent` menu
* adopt GIT and sync conflicts usage
* use content menu for "open in 3wm" command
* add new `ctxIsMergeResultEditor` context key

Better fix for https://github.com/microsoft/vscode/issues/153800

### CODE REVIEW

File: src/vs/workbench/browser/codeeditor.ts [hunk 1]
```diff
@@ -212,6 +216,44 @@ export class FloatingClickWidget extends Widget implements IOverlayWidget {
...
+			menuDisposables.clear();
+			if (!editor.hasModel() || editor.getOption(EditorOption.inDiffEditor)) {
+				return;
+			}
+			const actions: IAction[] = [];
+			createAndFillInActionBarActions(menu, { renderShortTitle: true, shouldForwardArgs: true }, actions);
+			if (actions.length === 0) {
+				return;
+			}
+			// todo@jrieken find a way to handle N actions, like showing a context menu
```
Comment (gjsjohnmurray) [inline src/vs/workbench/browser/codeeditor.ts:243]: Maybe use the kind of button-with-dropdown that the SCM view has.

File: src/vs/workbench/services/actions/common/menusExtensionPoint.ts [hunk 2]
```diff
@@ -280,10 +280,10 @@ const apiMenus: IAPIMenu[] = [
 		proposed: 'inlineCompletionsAdditions'
 	},
 	{
-		key: 'merge/toolbar',
-		id: MenuId.MergeToolbar,
-		description: localize('merge.toolbar', "The prominent botton in the merge editor"),
-		proposed: 'contribMergeEditorToolbar'
+		key: 'editor/content',
+		id: MenuId.EditorContent,
+		description: localize('merge.toolbar', "The prominent botton in an editor, overlays its content"),
```
Comment (gjsjohnmurray) [inline src/vs/workbench/services/actions/common/menusExtensionPoint.ts:285]:
```suggestion
		description: localize('merge.toolbar', "The prominent button in an editor, overlays its content"),
```

**Review by alexr00 — DISMISSED** [review]

File: src/vs/workbench/browser/codeeditor.ts (hunk 1 above)
Comment (jrieken) [reply src/vs/workbench/browser/codeeditor.ts:243]: yeah, something along those lines. But not for now, for later

**Comment by jrieken** [comment]
@alexr00 fixed typo, please re-review delta

**Review by alexr00 — APPROVED** [review]

**Pull request merged**